seaworthy/requirements.txt

```diff
@@ -0,0 +1 @@
+seaworthy[pytest] >= 0.1.1
```

Comment:
You probably want `>= 0.3.0` here. :-)

Reply:
Ah, the seaworthy-demo is a bit out of date :)